_maintenance_update_graph_data.sh.example

```diff
@@ -1,9 +1,10 @@
-#!/bin/bash
-set -o errexit
+#!/bin/sh
```

Comment:
Have you confirmed that this script doesn't contain any bash-isms? For instance, does `set -eu` work properly in `sh`?

Reply:
Yes, `set -eu` is correct in POSIX shell. I also tested the scripts using ShellCheck.